doc-content/enterprise-only/openshift/architecture-authoring-con.adoc

```diff
@@ -0,0 +1,92 @@
+[id='architecture-authoring-con-{context}']
+= Architecture of an authoring environment
+
+In {PRODUCT}, the {CENTRAL} component provides a web-based interactive user interface for authoring services. The {KIE_SERVER} component runs the services. 
+
+ifdef::PAM[]
+The {KIE_SERVER} uses a database server to store the state of process services. 
+endif::PAM[]
+
+
+You can also use {CENTRAL} to deploy the services that you develop onto a {KIE_SERVER}.  You can use several {KIE_SERVERS} to run different services and control the servers from the same {CENTRAL}.
+
+[float]
+== Single authoring environment
+
+In a single authoring environment, only one instance of {CENTRAL} is running. Multiple users can access its web interface at the same time, however the performance can be limited and there is no failover capability.
+
+{CENTRAL} includes a built-in Maven repository that stores the built versions of the services that you develop (KJAR files/artifacts). You can use your own continuous integration and either continuous deployment (CICD) tools to retrieve these artifacts from the repository and move them as necessary.
+
+{CENTRAL} stores the source code in a built-in Git repository, stored in the `.niogit` directory. It uses a built-in indexing mechanism to index the assets in your services.
+
+{CENTRAL} uses persistent storage for the Maven repository and for the Git repository.
+
+A single authoring environment, by default, includes one {KIE_SERVER}. 
+ifdef::PAM[]
+This {KIE_SERVER} uses a built-in H2 database engineto store the state of process services. 
+endif::PAM[]
+
+ifeval::["{context}"!="openshift-operator"]
+A single authoring environment, by default, uses the _controller strategy_.
+endif::[]
+ifeval::["{context}"=="openshift-operator"]
+A single authoring environment can use the _controller strategy_.
+endif::[]
+{CENTRAL} includes the _Controller_, a component that can manage {KIE_SERVERS}. When you configure a {KIE_SERVER} to connect to {CENTRAL}, the {KIE_SERVER} uses a REST API to connect to the Controller. This connection opens a persistent WebSocket. In an OpenShift deployment that uses the controller strategy, each {KIE_SERVER} is initially configured to connect to the {CENTRAL} Controller. 
+
+When you use the {CENTRAL} user interface to deploy or manage a service on the {KIE_SERVER}, the {KIE_SERVER} receives the request through the Controller connection WebSocket. To deploy a service, the {KIE_SERVER} can request the necessary artifact from the Maven repository that is a part of {CENTRAL}. 
```

Comment:
Suggested rewrite:

... To deploy a service, the {KIE_SERVER} requests the ...